### PR TITLE
PORTALS-2688: first pass with Lighthouse accessibility results

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: packages/synapse-react-client/src/components/SynapseNavDrawer.tsx [hunk 1]
```diff
@@ -137,15 +137,17 @@ const NavDrawerListItem = (props: MenuItemParams) => {
...
 
   return onClickGoToUrl ? (
-    <a
-      href={onClickGoToUrl}
-      rel="noopener noreferrer"
-      className="SRC-whiteText"
-    >
-      {listItem}
-    </a>
+    <li>
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/components/SynapseNavDrawer.tsx:140]: https://dequeuniversity.com/rules/axe/4.6/list

File: packages/synapse-react-client/src/components/SynapseNavDrawer.tsx [hunk 2]
```diff
@@ -231,7 +233,12 @@ export const SynapseNavDrawer: React.FunctionComponent<
         className={`SynapseNavDrawerMenu ${isOpen ? 'tempDrawerOpen' : ''}`}
       >
         <div onClick={handleDrawerClose}>
-          <a className="synapseIcon" rel="noopener noreferrer" href="/#!Home:0">
+          <a
+            className="synapseIcon"
+            rel="noopener noreferrer"
+            href="/#!Home:0"
+            aria-label="Synapse Home"
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/components/SynapseNavDrawer.tsx:240]: https://dequeuniversity.com/rules/axe/4.6/link-name

File: packages/synapse-react-client/src/components/menu/ComplexMenu.tsx [hunk 3]
```diff
@@ -48,6 +48,7 @@ export function ComplexMenu(props: ComplexMenuProps) {
                 color="primary"
                 disabled={iconButton.disabled}
                 onClick={iconButton.onClick}
+                aria-label={iconButton.tooltipText}
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/components/menu/ComplexMenu.tsx:51]: https://dequeuniversity.com/rules/axe/4.6/link-name

File: packages/synapse-react-client/src/style/abstracts/_variables.scss [hunk 4]
```diff
@@ -10,7 +10,7 @@ $colors: (
   'gray-1000': #22252a,
   'gray-900': #353a3f,
   'gray-800': #4a5056,
-  'gray-700': #878e95,
+  'gray-700': #6f7478,
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/style/abstracts/_variables.scss:13]: https://dequeuniversity.com/rules/axe/4.6/color-contrast

Before (light grey on white), failed AA:
Screen Shot 2023-06-07 at 1 48 35 PM
After (slightly darker light grey on white), passes AA:
Screen Shot 2023-06-07 at 1 48 16 PM

**Comment by jay-hodgson** [comment]
(seems to be an unrelated error in synapse-react-client integration tests : test/containers/entity/annotations/SchemaDrivenAnnotationsEditor.integration.test.tsx)

File: packages/synapse-react-client/src/style/abstracts/_variables.scss [hunk 5]
```diff
@@ -10,7 +10,7 @@ $colors: (
   'gray-1000': #22252a,
   'gray-900': #353a3f,
   'gray-800': #4a5056,
-  'gray-700': #878e95,
+  'gray-700': #6e757c,
```
Comment (nickgros) [inline packages/synapse-react-client/src/style/abstracts/_variables.scss:13]: https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA?node-id=213:7173#466693261

**Review by nickgros — APPROVED** [review]

**Pull request merged**